go/streamlog/streamlog.go

```diff
@@ -42,7 +42,10 @@ var (
 	QueryLogFormat = flag.String("querylog-format", "text", "format for query logs (\"text\" or \"json\")")
 
 	// QueryLogFilterTag contains an optional string that must be present in the query for it to be logged
-	QueryLogFilterTag = flag.String("querylog-filter-tag", "", "string that must be present in the query for it to be logged")
+	QueryLogFilterTag = flag.String("querylog-filter-tag", "", "string that must be present in the query for it to be logged; if using a value as the tag, you need to disable query normalization")
+
+	// QueryLogRowLimit only log queries returning or affecting this many rows
+	QueryLogRowLimit = flag.Uint64("querylog-row-limit", 0, "Minimum number of rows a query has to return or affect before being logged; not useful for streaming queries")
```

Comment:
It's common for "limit" type params to also have "unlimited" value. e.g. `0` or `-1` would mean "unlimited", and positive numbers would impose the minimal number of rows. Currently the code does not support unlimited values. What do you think?

Reply:
Good point.  There are no similar "limit" variables with "unlimited" values.  I will adjust

Reply:
OK, I thought about it;  and it makes sense to just rename the parameter to reflect that it's a threshold and not a limit.  Other than this name change;  no real other functional changes.